packages/webapi/models/core_accounts.go

```diff
@@ -2,36 +2,14 @@ package models
 
 import "github.com/iotaledger/wasp/packages/isc"
 
-type AccountsResponse struct {
-	AccountIDs []string `json:"accountIds" swagger:"required"`
-}
-
-type AccountListResponse struct {
-	Accounts []string `json:"accounts" swagger:"required"`
-}
-
 type AssetsResponse struct {
-	BaseTokens   string         `json:"baseTokens" swagger:"required,desc(The base tokens (uint64 as string))"`
-	NativeTokens []isc.CoinJSON `json:"nativeTokens" swagger:"required"`
-}
-
-type AccountNFTsResponse struct {
-	NFTIDs []string `json:"nftIds" swagger:"required"`
+	Coins []isc.CoinJSON `json:"coins" swagger:"required"`
 }
 
-type AccountFoundriesResponse struct {
-	FoundrySerialNumbers []uint32 `json:"foundrySerialNumbers" swagger:"required"`
+type AccountObjectsResponse struct {
+	ObjectIDs []string `json:"objectIds" swagger:"required"`
```

Comment:
We have access to the object type. I think it would be good to add that as well.